core/cap-0035.md

```diff
@@ -211,17 +220,7 @@ index 7f08d757..8a55d046 100644
      }
      body;
  };
-@@ -915,7 +956,8 @@ enum AllowTrustResultCode
-                                     // source account does not require trust
-     ALLOW_TRUST_TRUST_NOT_REQUIRED = -3,
-     ALLOW_TRUST_CANT_REVOKE = -4,     // source account can't revoke trust,
--    ALLOW_TRUST_SELF_NOT_ALLOWED = -5 // trusting self is not allowed
-+    ALLOW_TRUST_SELF_NOT_ALLOWED = -5, // trusting self is not allowed
-+    ALLOW_TRUST_CANT_CHANGE_FLAG_CLAWBACK_ENABLED = -6 // changing the clawback enabled flag not allowed

```

Comment:
This result code is no longer necessary.

Reply:
It's a little confusing from the line this comment is on which flag you were referring to. The flag you this line is already removed. However while trying to find it I see I had left `CREATE_CLAIMABLE_BALANCE_CLAWBACK_REQUIRED` which is no longer required either, so I removed it in 936bd5d.

Reply:
Ah my bad you're right.